nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/jwt/JwtAuthenticationFilter.java

```diff
@@ -33,43 +32,25 @@
     private static final Logger logger = LoggerFactory.getLogger(JwtAuthenticationFilter.class);
 
     // The Authorization header contains authentication credentials
-    public static final String AUTHORIZATION = "Authorization";
-    private static final Pattern tokenPattern = Pattern.compile("^Bearer (\\S*\\.\\S*\\.\\S*)$");
+    public static final String JWT_COOKIE_NAME = "__Host-jwt-auth-cookie";
```

Comment:
Does this need to be public?

Reply:
It is used in a few places, I'm not sure how best to share this value.

Reply:
Recommend removing `cookie` from the name and changing it to something more generic.
```suggestion
    public static final String JWT_COOKIE_NAME = "__Host-Authorization-Bearer";
```